Gemfile.lock

```diff
@@ -330,3 +330,6 @@ DEPENDENCIES
   uk_postcode (~> 1.0.1)
   unicorn (= 4.8.3)
   webmock (= 1.20.4)
+
+BUNDLED WITH
+   1.10.6
```

Comment:
I think we're going to have to live with the `BUNDLED WITH` line being added to Gemfile.lock sooner or later, so we may as well have it in this commit.

I did just try running `bundle install` in master to see whether that would add `BUNDLED WITH` to the Gemfile.lock, but it didn't. Presumably it only gets added when something actually changes, i.e. a Gem version bump like this.